apax/data/input_pipeline.py

```diff
@@ -44,6 +46,7 @@ def __init__(
         n_jit_steps=1,
         pre_shuffle=False,
         ignore_labels=False,
+        cache_path=".",
```

Comment:
cache path maybe better in experiment dir?

Reply:
It is set to the experiment dir in run.py 
This is just the default, but we can remove that if you prefer.

Reply:
No its finde.